docs/src/maintain/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How ESLint is Maintained
+eleventyNavigation:
+    key: how eslint is maintained
+    parent: maintain eslint
+    title: How ESLint is Maintained
+    order: 1
+
+---
+
+This page explains the different roles and structures involved in maintaining ESLint.
+
+## The ESLint Team
+
+The ESLint team work together to develop and maintain ESLint. To learn more about the different roles on the ESLint team, refer to [Governance](../contribute/governance). To see the current team members, refer to [Team](https://eslint.org/team/).
```

Comment:
I'd prefer teams page to be redirected to the actual host it is running on instead of always redirecting to eslint.org site. WDYT?

Reply:
```suggestion
The ESLint team work together to develop and maintain ESLint. To learn more about the different roles on the ESLint team, refer to [Governance](../contribute/governance). To see the current team members, refer to [Team](/team).
```

is this what you mean?

Reply:
Yes